charms/kfp-viewer/contributing_inputs.yaml

```
@@ -0,0 +1,19 @@
+charm: "kfp-viewer"
+tox_example_dev_env: "lint"
+os_base: "ubuntu-20.04"
+deploy_opts: |-
+  \
+    --resource oci-image=$(yq '.resources."oci-image"."upstream-source"' metadata.yaml)
```

See: https://github.com/canonical/kfp-operators/blob/38fd0f0d8ae5a6195930add44543700986bfc891/charms/kfp-viewer/metadata.yaml#L11-L15
```suggestion
    --resource oci-image=$(yq '.resources."kfp-viewer-image"."upstream-source"' metadata.yaml)
```

---

Then please rerender `contributing.md`.